internal/database/online/tikv/util.go

```diff
@@ -4,6 +4,14 @@ import "strings"
 
 const keyDelimiter = ":"
 
-func getKey(revisionID string, entityKey string, featureID string) []byte {
-	return []byte(strings.Join([]string{revisionID, entityKey, featureID}, keyDelimiter))
+func getKeyOfBatchFeature(revisionID string, entityKey string, featureID string) []byte {
+	return []byte(keyPrefixForBatchFeature + getKey(revisionID, entityKey, featureID))
+}
+
+func getKeyOfStreamFeature(groupID string, entityKey string, featureID string) []byte {
+	return []byte(keyPrefixForStreamFeature + getKey(groupID, entityKey, featureID))
```

Comment:
why don't just use `kvutil.KeyPrefixForStreamFeature`?

Reply:
Similar to backend

Reply:
fixed